test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -359,31 +360,72 @@ function m.updatePreloadedFile(pUpdateFunc)
   m.setPreloadedPT(pt)
 end
 
-function m.processGetVDsuccess(pData)
+function m.processGetVDsuccess(pData, pHmiResParams)
+  pHmiResParams = pHmiResParams or m.allVehicleData[pData]
+
+  local hmiResParams = {
+    [pData] = pHmiResParams.value
+  }
   local reqParams = {
-     [pData] = true
+    [pData] = true
+  }
+
+  m.getVehicleDataSuccessExpectation(reqParams, reqParams, hmiResParams)
+end
+
+function m.processGetVDsuccessManyParameters( ... )
+  local reqParams = {}
+  local hmiResParams = {}
+
+  for _, vehicleData in ipairs( { ... } ) do
+    reqParams[vehicleData] = true
+    hmiResParams[vehicleData] = m.allVehicleData[vehicleData].value
+  end 
+
+  m.getVehicleDataSuccessExpectation(reqParams, reqParams, hmiResParams)
+end
+
+function m.processGetVDsuccessCutDisallowedParameters(pDisallowedData, pAllowedData)
+  local reqParams = {
+    [pDisallowedData] = true,
+    [pAllowedData] = true
+  }
+  local expectReqParams = {
+    [pAllowedData] = true
   }
   local hmiResParams = {
-    [pData] = m.allVehicleData[pData].value
+    [pAllowedData] = m.allVehicleData[pAllowedData].value
   }
-  local cid = m.getMobileSession():SendRPC("GetVehicleData", reqParams)
-  m.getHMIConnection():ExpectRequest("VehicleInfo.GetVehicleData", reqParams)
+
+  local info = '\'' .. pDisallowedData .. '\'' ..  " disallowed by policies."
```

Comment:
@ydementieiev `...is disallowed by policies`

Reply:
@AByzhynar in sdl_core we have` info += " disallowed by policies.";` if I change to your variant we will have failed test